packages/search/src/elasticsearch/client.ts

```diff
@@ -53,3 +53,13 @@ export interface ISearchResponse<T> {
 export const client = new elasticsearch.Client({
   node: `http://${ES_HOST}`
 })
+
+export const getOrCreateClient = () => {
```

Comment:
If it's okay I would like to remove the client created immediately above but rather use a getter to ensure single instance and easier mocking.